### ISSUE TITLE
No order in development process

### ISSUE THREAD

**Issue by BirgitBoss** [OpenManufacturingPlatform/rules_of_engagement#23]
Do not include any order in which documents need to be created. 
Technical Development "The first document to be developed is" ==> remove, it is sufficient to have a classification of documents. Just state the process of any document.

the only interesting topic would be: IF there is a formal requirements document: how to do traceability in technical specifications...

**Comment by jpradocueva** [comment]
- [ ] PR needed to remove "The first document to be developed is"

**Comment by jpradocueva** [comment]
@BirgitBoss 
> the only interesting topic would be: IF there is a formal requirements document: how to do traceability in technical specifications...

... by traceability are you referring to establish a correlation of each requirement with its technical solution? If yes, it can be done but it requires a bit of discipline and will slow down the development process. 

Do you think that it may be necessary?

**Comment by jpradocueva** [comment]
See PR #27

**Comment by BirgitBoss** [comment]
Yes, I really think there needs to be a very simple template for traceability. I.e. a template for an Req.-Document that numberes Requirements and a design-document template with a table that references these Requirements. However, this is not necfessary for the first version of templates for OMP. But in the long run I know from other organizations that this will be needed.

**Comment by jpradocueva** [comment]
I can provide a template that is used in other standards organizations.

On your comment:
> However, this is about templates, not about engagement rules ...

You are right, this comment doesn't need to be link to PR #27. This means that I did apply the reference incorrectly, ... there is an issue somewhere that it is missing the reference to PR #27

**Comment by jpradocueva** [comment]
@BirgitBoss https://github.com/OpenManufacturingPlatform/templates/pull/10  is a proposal to introduce static conformance requirements as per our discussion.
Will this table satisfy your requirements?
